docs/UserGuide.md

```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 
 ---
 
-### Sorting the shows
-[TO CHANGE AFTER REWORK]
+### Sorting the shows `sort`
 
-Sort shows based on the input prefix.
+Your show list is in mess! Do you need a way to organise the list?
+Sort can help reorder the list!!
 
-Format: `sort [sna/] [snd/] [ssa/] [ssd/] [so/]…​`
-* Use sna/ to sort name in ascending order.
-* Use snd/ to sort name in descending order.
-* Use ssa/ to sort status in ascending order.
-* Use ssd/ to sort status in descending order.
-* If there are no prefix, it will sort by name in ascending order.
-* If both prefixes for ascending and descending are used for the same attribute, it will only sort by ascending.
-* If both prefixes for name and status are used, by default, it will sort by name then status.
-* Use so/ to sort by status then name.
+:exclamation:**Caution:**<br>
+*You cannot undo the sort once you applied it!
+
+Format: `sort [n/ORDER] [s/ORDER] [t/ORDER] [r/ORDER] [so/SEQUENCE]…​`
+
+`Sort` will sort according to the [prefixes](#command-structure)**
+* Use `n/` if you want to sort by name 
+* Use `s/` if you want to sort by status
+* Use `t/` if you want to sort by tag
+* Use `r/` if you want to sort by rating
+* For the above prefixes, you have to specify the `ORDER` right after the prefix.
+* `ORDER` is the order you want to sort the list by! Type `asc` to sort by ascending or type `dsc` to sort by descending! 
+* If two or more of the above prefixes are being used, sort will prioritise sorting by name then status then rating then tag.   
+* Use `so/` if you want to reorder the priority and state the `SEQUENCE`
+* List the `SEQUENCE` by listing out the full name of the prefix used in the order you want.
+* For example: you want to sort by rating then tag then status then name, `SEQUENCE` will be "RatingTagStatusName"
+* If you are confused, don't worry! Just do not use any prefix, we will help you sort it by name in ascending order!  
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about Sort:**<br>
+* By not entering any prefixes, it will sort by name in ascending order.
+* Enter the right amount prefixes! For example: if you only sort by status, use only `s/`. 
+* The `ORDER` must be asc or dsc!
+* The `SEQUENCE` must contain the full name of all the prefixes used!
+* For `SEQUENCE`, if more than one of the same full name is being used, it will only consider the first instance of the full name. For example: "RatingTagRating" will sort by rating then tag. 
```

Comment:
Maybe we can change full name to prefix? This will follow the way you address them in the next line.
I'd like to slightly rephrase the first sentence to
```
For `SEQUENCE`, if the same prefix is being used multiple times, it will only consider the first instance of the prefix.
```
Might just be me, but I feel that this will make it easier for the reader to understand the meaning of the sentence.

Reply:
If you use prefix it can refer to "so/" too, might cause further confusion?

Reply:
Ahh that's true. Maybe it'd be more correct to use the term sort conditions?
`if a sort condition is being used multiple times` instead? Would this help?

Reply:
ok